### PR TITLE
docs: fix the issue of incorrect links to referenced documents in the project

### PR DESCRIPTION
**Pull request by hansedong**
Fix issue: the main issue addressed was the incorrect document links referenced throughout the project.

### CODE REVIEW

**Comment by faisalburhanudin** [comment]
hi @hansedong, maybe you can include changes for all markdown links ?

such as
- quick-start.MD
- backups.MD

**Comment by hansedong** [comment]
@faisalburhanudin 
In the project, I have also found and fixed the issue of incorrect document links referenced elsewhere, please kindly review again.
However, what I need to point out is that there is a reference to `/docs/backups.md` in the `README.md` document. Although I changed `backups.MD` to `backups.md`, I did not find any files related to backups.md in the project. What should this place be changed to?

File: README.md [hunk 1]
```diff
@@ -18,18 +18,18 @@ So you can use familiar CRD objects: `ServiceMonitor`, `PodMonitor`, `Prometheus
...
 
-Besides, operator allows you to manage VictoriaMetrics applications inside kubernetes cluster and simplifies this process [quick-start](/docs/quick-start.MD)
-With CRD (Custom Resource Definition) you can define application configuration and apply it to your cluster [crd-objects](/docs/api.MD).
+Besides, operator allows you to manage VictoriaMetrics applications inside kubernetes cluster and simplifies this process [quick-start](/docs/quick-start.md)
+With CRD (Custom Resource Definition) you can define application configuration and apply it to your cluster [crd-objects](/docs/api.md).
 
  Operator simplifies VictoriaMetrics cluster installation, upgrading and managing.
 
- It has integration with VictoriaMetrics `vmbackupmanager` - advanced tools for making backups. Check backup [docs](/docs/backups.MD)
+ It has integration with VictoriaMetrics `vmbackupmanager` - advanced tools for making backups. Check backup [docs](/docs/backups.md)
```
Comment (Haleygo) [inline README.md]:
```suggestion
 It has integration with VictoriaMetrics [vmbackupmanager](https://docs.victoriametrics.com/vmbackupmanager.html) - advanced tools for making backups. Check [Backup automation for VMSingle](./resources/vmsingle.md#backup-automation) or [Backup automation for VMCluster](./resources/vmcluster.md#backup-automation).```

**Review by Haleygo — CHANGES_REQUESTED** [review]
Thanks for the fixes, plz see the suggestion.

**Comment by hansedong** [comment]
@Haleygo Updated, thanks for the suggestion.

**Review by Haleygo — APPROVED** [review]
LGTM!

**Pull request merged**